tests/bot/exts/moderation/test_incidents.py

```diff
@@ -768,3 +787,73 @@ async def test_on_message_non_incident(self):
             await self.cog_instance.on_message(MockMessage())
 
         mock_add_signals.assert_not_called()
+
+
+class TestMessageLinkEmbeds(TestIncidents):
+    """Tests for `extract_message_links` coroutine."""
+
+    async def test_shorten_text(self):
+        """Test all cases of text shortening by mocking messages."""
+        tests = {
+            "thisisasingleword"*10: ('thisisasingleword'*10)[:50]+"...",
+            "\n".join("Lets make a new line test".split()): "Lets\nmake\na"+"...",
+            'Hello, World!' * 300: ('Hello, World!' * 300)[:300] + '...'
+        }
```

Comment:
This is really difficult to read. Could we figure out a better way? Verbosity is sometimes worth it in tests, you want the cases to be obvious to whoever will have to read them if something fails.

The code seems to mix single and double quotes and doesn't seem to have spaces around some operators, is that intended?

Reply:
Do you have ideas for making it more readable.

No, that is not intended behavior, I will change them.

Reply:
I'd rather these were string literals without the extra processing, even if it takes space. Just clear cases of input and output.